src/main/java/com/nerodesk/om/Account.java

```diff
@@ -41,6 +42,24 @@
 @Immutable
 public interface Account {
     /**
+     * Dummy implementation.
+     */
+    Account DUMMY = new Account() {
```

Comment:
But on the other hand we have only one instance of the dummy class, so less memory usage.
I'm not sure about this, both options have they pros and cons.